app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -139,13 +148,23 @@ class WireActivityViewModel @Inject constructor(
         observeUpdateAppState()
         observeNewClientState()
         observeScreenshotCensoringConfigState()
+        observeAccounts()
+    }
+
+    private fun observeAccounts() {
+        viewModelScope.launch {
+            observeValidAccounts().distinctUntilChanged().collectLatest {
+                updateLoggedInUsersCount(it.size)
```

Comment:
this looks like something that could be done in kalium side only. What's the point of having it in AR, user doesn't affect it anyway. 

We have some examples in `UserSessionScope` in the `init` blick we launch the coroutines that are supposed to run in background without users impact.